dozer/db.py

```diff
@@ -1,17 +1,20 @@
+"""Provides database storage for the Dozer Discord bot"""
+
 import sqlalchemy
 from sqlalchemy import Column, Integer, String, Boolean, ForeignKey, ForeignKeyConstraint
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, Session, sessionmaker
 
 __all__ = ['engine', 'DatabaseObject', 'Session', 'Column', 'Integer', 'String', 'ForeignKey', 'relationship',
-           'Boolean']
+           'Boolean', 'ForeignKeyConstraint']
 
 engine = sqlalchemy.create_engine('sqlite:///dozer.db')
 DatabaseObject = declarative_base(bind=engine, name='DatabaseObject')
 DatabaseObject.__table_args__ = {'extend_existing': True}  # allow use of the reload command with db cogs
 
 
 class CtxSession(Session):
+    """Provides a context session for the database system."""
```

Comment:
Specifically, this allows sessions to be used as context managers and asynchronous context managers.